x-pack/plugins/cases/public/components/recent_cases/index.test.tsx

```diff
@@ -91,12 +104,21 @@ describe('RecentCases', () => {
     });
 
     // apply the filter
-    const myRecentCasesElement = getByTestId('myRecentlyReported');
-    userEvent.click(myRecentCasesElement);
+    await waitFor(() => {
+      const myRecentCasesElement = getByTestId('myRecentlyReported');
+      userEvent.click(myRecentCasesElement);
+    });
 
     expect(useGetCasesMock).toHaveBeenLastCalledWith({
       filterOptions: {
-        reporters: [{ email: undefined, full_name: undefined, username: undefined }],
```

Comment:
these were undefined before because the mock was returning a promise when the hook actually didn't return a promise.